### PR TITLE
runtools enhancements & build script for analyzer container

### PR DESCRIPTION
**Pull request by AOrps**
- Changed how argument parsing works 
  - included a usage function (will flesh it out more options later)
  - will pick the first instance of a `pkg:<pkg_manager>/<pkg>@<version_num>` as opposed to the first parameter
- Changed blue to bright cyan on terminal colors (went from being invisible on dark terminals to readable)
- dynamic version resolution: 
- added a build script that (tries to closely) mirror the build.ps1 functionality for the analyzer container
- updated toolshed version number

### CODE REVIEW

**Comment by AOrps** [comment]
Needed to force-push to signoff on all commits to be compliant with DCO

File: omega/analyzer/build.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+	if [ $OPT_FORCE ]; then
+	    docker build -t openssf/omega-toolshed:$version . -f Dockerfile --build-arg CACHEBUST=$(date '+%FT%T.%N%:z')
+	else
+	    docker build -t openssf/omega-toolshed:$version . -f Dockerfile
+	fi
+	docker tag openssf/omega-toolshed:$version openssf/omega-toolshed:latest
+	
+    } || { #catch
+	echo "Error running build."
```
Comment (Cyber-JiuJiteria) [inline omega/analyzer/build.sh:32]: Anyway we can include any error messages from the failure? This will help with troubleshooting.

**Review by Cyber-JiuJiteria — APPROVED** [review]

File: omega/analyzer/build.sh (hunk 1 above)
Comment (AOrps) [reply omega/analyzer/build.sh:32]: Are you talking about failure as a result of the build?

Comment (AOrps) [reply omega/analyzer/build.sh:32]: --> buff out the script to catch errors on any part of the script where it may fail
+ log statements, make it unique

**Review by Cyber-JiuJiteria — APPROVED** [review]

**Review by Cyber-JiuJiteria — COMMENTED** [review]
compelted review. merging

**Pull request merged**